src/context/WalletProvider/config.ts

```diff
@@ -103,4 +106,11 @@ export const SUPPORTED_WALLETS: Record<KeyManager, SupportedWalletInfo> = {
     ...DemoConfig,
     routes: [],
   },
+  [KeyManager.Keplr]: {
+    ...KeplrConfig,
+    routes: [
+      { path: '/Keplr/connect', component: KeplrConnect },
+      { path: '/Keplr/failure', component: KeplrFailure },
```

Comment:
Inconsistent casing with the rest of the module
```suggestion
      { path: '/keplr/connect', component: KeplrConnect },
      { path: '/keplr/failure', component: KeplrFailure },
```

Routes are mapped dynamically here:
https://github.com/shapeshift/web/blob/22272912f25a792f26c788bc1548c77df9337c4e/src/context/WalletProvider/WalletViewsSwitch.tsx#L96

So it should be possible to use the same lowercase

Reply:
👍